### PR TITLE
Prohibit the collection and sharing of member info

### PR DESCRIPTION
**Pull request by MarkBennett**
Encourage members to treat personal information with the privacy and dignity.

Prohibit the collection and sharing of personal information without prior and explicit consent.

Clarify that the Code of Conduct applies to online venues like Slack, emails, and forums.

### CODE REVIEW

File: code_of_conduct.md [hunk 1]
```diff
@@ -47,6 +48,7 @@ The following behaviors are considered harassment and are unacceptable within ou
 *   Deliberate intimidation, stalking or following (online or in person).
 *   Advocating for, or encouraging, any of the above behavior.
 *   Sustained disruption of community events, including talks and presentations.
+*   Collecting or sharing personal information (including names, emails, nicks, messages, emails, or other biographical or identifying information) without explicit and prior consent.
```
Comment (abramhindle) [inline code_of_conduct.md]: This is very broad and includes logging or just caching.

Comment (MarkBennett) [reply code_of_conduct.md]: The Code of Conduct applies to people communicating over DES media and venues, as well as attending DES supported events, not DES itself, so I believe the society could still operate our own logs, etc. Specifically, we're trying to prohibit people from signing up and mining the Slack channels as we saw that happen on a few other Slacks here in town.

Any suggestions on making this clearer?

In terms of DES' use of the data, a privacy policy would probably be the place to do that. I've setup #19 to remember to do this.

Comment (nathany) [reply code_of_conduct.md]: Should "nicks" be "nicknames"

File: code_of_conduct.md [hunk 2]
```diff
@@ -68,7 +70,7 @@ Additionally, community organizers are available to help community members engag
 
 ## 7. Scope
 
-We expect all community participants (people, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues–online and in-person–as well as in all one-on-one communications pertaining to community business.
+We expect all community participants (people, paid or otherwise; sponsors; and other guests) to abide by this Code of Conduct in all community venues–online and in-person–as well as in all one-on-one communications pertaining to community business. This Code of Conduct applies online (including Slack, email, forums) and offline.
```
Comment (nathany) [inline code_of_conduct.md]: maybe the more general "chat" instead of "Slack"?

Comment (MarkBennett) [reply code_of_conduct.md]: Good call. Updating..

**Comment by MarkBennett** [comment]
@nathany @abramhindle any more changes? Can you approve if you're satisfied and I'll merge.

**Review by nathany — APPROVED** [review]
Tweet others as you'd like to be tweeted. 👍

File: code_of_conduct.md [hunk 3]
```diff
@@ -47,6 +48,7 @@ The following behaviors are considered harassment and are unacceptable within ou
 *   Deliberate intimidation, stalking or following (online or in person).
 *   Advocating for, or encouraging, any of the above behavior.
 *   Sustained disruption of community events, including talks and presentations.
+*   Collecting or sharing personal information (including names, emails, avatars, handles, nicknames, messages, emails, or other biographical or identifying information) without explicit and prior consent.
```
Comment (nathany) [inline code_of_conduct.md:51]: ohh.. "emails" is twice

Comment (nathany) [reply code_of_conduct.md:51]: also, maybe it should be "email addresses"?

Comment (MarkBennett) [reply code_of_conduct.md:51]: I’m out but will fix this @nathany. Good catch!

**Pull request merged**